src/LevelDBStore/LevelDBStore/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "PluginConfiguration": {
+    "Network": 5195086
```

Comment:
We don't need this. All networks need storage plugins.

Reply:
so should the paths of all plugins be network-independent? Or just the plugin using `NeoSystem` be network-dependent?